funds-manager/funds-manager-server/src/handlers.rs

```diff
@@ -67,8 +68,8 @@ pub(crate) async fn get_deposit_address_handler(
     query_params: HashMap<String, String>,
     server: Arc<Server>,
 ) -> Result<Json, warp::Rejection> {
-    let mint = query_params.get(MINT_QUERY_PARAM).ok_or_else(|| {
-        warp::reject::custom(ApiError::BadRequest("Missing 'mint' query parameter".to_string()))
+    let mint = query_params.get(MINTS_QUERY_PARAM).ok_or_else(|| {
+        warp::reject::custom(ApiError::BadRequest("Missing 'mints' query parameter".to_string()))
```

Comment:
pedantic nit: I would keep this parameter as the singular `mint` to make the API more readable (at the expense of an extra constant)

Reply:
disagree that it's more readable, i'll keep it this way. this is a very common pattern to accept list arguments